env_config.rb

```diff
@@ -44,10 +44,15 @@
 
     # Local-specific stuff
     optional :ENABLE_BULLET, :bool, false
-    optional :SKIP_PRETEST_SETUP, :bool, false
     optional :MAILCATCHER_SMTP_HOST, :string, ''
   end
 
+  if Rails.env.test?
+    optional :DISABLE_WEBMOCK, :bool, false
+    optional :ENABLE_NET_CONNECT_IN_TESTS, :bool, true
```

Comment:
Here you declared it as `ENABLE` but below in the actual config you're using `DISABLE`